servicetalk-gradle-plugin-internal/src/main/groovy/io/servicetalk/gradle/plugin/internal/ServiceTalkLibraryPlugin.groovy

```diff
@@ -233,6 +233,11 @@ final class ServiceTalkLibraryPlugin extends ServiceTalkCorePlugin {
         // Always require native libraries for running tests. This helps to make sure transport-level state machine
         // receives all expected network events from Netty.
         systemProperty "io.servicetalk.transport.netty.requireNativeLibs", "true"
+
+        // Make sure we enable netty leak detection for our tests. By default these are not visible in the logs,
+        // to see them you must at --warn to your ./gradlew test run in order to see the "showStandardStreams".
+        systemProperty "io.netty.leakDetection.level", "PARANOID"
```

Comment:
One of the main goals is to check for leaks automatically via `ci-prb.yml` pipeline. There we can not simply add `--warn` because it will result in extremely long unreadable build output. We reduced `events` down to `[FAILED]` because of that. 

We should find a way to always be `PARANOID` on CI with output to standard streams without changing `events`, while local development should be more like opt-in for leak detection output as a way to troubleshoot when CI finds something.

One potential idea is to change `showStandardStreams` logic to be a check on `io.netty.leakDetection.level`. If it's `PARANOID`, then it results to `true`. And then add a system property in `ci-prb.yml`. Maybe there is other ways as well.